index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+    <style>
+        table {
+
+			border-collapse: collapse;
+		}
+
+		thead tr{
+			border-top: dashed blue 2px;
+			border-bottom: dashed blue 2px;
+
+		}
+
+		tbody:before{
```

Comment:
why ?

Reply:
to get a space between the thead and the tbody (not the best way I know, but margins were not working)